packages/nuxt/tsconfig.json

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "./.nuxt/tsconfig.json"
+  "extends": "./.nuxt/tsconfig.json",
+  "compilerOptions": {
+    "paths": {
+      "@storefront/shared": ["../shared"]
+    }
+  },
```

Comment:
`@storefront/shared` is package & devDependency - I believe that change isn't needed

Reply:
Didn't work otherwise :(